docs/content/deprecation/releases.md

```diff
@@ -6,6 +6,7 @@ Below is a non-exhaustive list of versions and their maintenance status:
 
 | Version | Release Date | Active Support     | Security Support | 
 |---------|--------------|--------------------|------------------|
+| 3.0     | Apr 29, 2024 | Yes                | Yes              |
 | 2.11    | Feb 12, 2024 | Yes                | Yes              |
```

Comment:
what is the date of the ended support of the v2?

Reply:
The version v2.11 is still maintained as it's the latest minor of the previous major.
I left `Yes` in the support columns as the end date is calculable according to the section in the same page: 

> - **The last `minor` after releasing a new `major`** will be supported for 1 year following the `major` release

However, adding the date to the table could be useful too.

Reply:
Can you update the table to have an explicit end date (Apr 29, 2025)?